### DISCUSSION TITLE
tmux: open terminal failed: can't find terminfo database

### DISCUSSION THREAD

**Discussion by jakobbg** [Homebrew/discussions#584] [Everyday usage]
MacBook Air M1: When running tmux I get this: ```open terminal failed: can't find terminfo database```. {sadface} How to resolve? :-)

**Comment by carlocab** [comment]
This doesn't sound like a Homebrew issue, so this isn't really the best place to ask for help about this. You should try the tmux mailing list.

That said, I'd check the output of `echo $TERM`, and make sure it says something sane, like `screen-256color`. It might help to set this in your `tmux.conf` using `set-option default-terminal screen-256color`.

**Reply by hyuraku** [reply]
> This doesn't sound like a Homebrew issue

I think so. Maybe the relevant issue is https://github.com/tmux/tmux/issues/743
